npm_and_yarn/lib/dependabot/npm_and_yarn/file_fetcher.rb

```diff
@@ -182,25 +182,36 @@ def inferred_npmrc # rubocop:disable Metrics/PerceivedComplexity
 
       sig { returns(T.nilable(T.any(Integer, String))) }
       def npm_version
-        @npm_version ||= T.let(package_manager.setup("npm"), T.nilable(T.any(Integer, String)))
+        @npm_version ||= T.let(package_manager_helper.setup("npm"), T.nilable(T.any(Integer, String)))
```

Comment:
do we have constants for `npm`, `pnpm`, and `yarn`?

Reply:
Yes, I created them. Will replace constants.

Reply:
solved